src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -144,13 +164,18 @@ public static IndividualDetails getPartyIndividualDetails(Entity entity, Hearing
     }
 
     public static String getIndividualFirstName(Entity entity) {
-        return entity.getName().getFirstName();
+        return nonNull(entity.getName()) ? entity.getName().getFirstName() : "";
     }
 
     public static String getIndividualLastName(Entity entity) {
-        return entity.getName().getLastName();
+        return nonNull(entity.getName()) ? entity.getName().getLastName() : "";
     }
 
+    public static String getIndividualFullName(Entity entity) {
+        return nonNull(entity.getName()) ? entity.getName().getFullName() : "";
```

Comment:
Name should not be empty, if it throws an error, that is okay as the caseData is malformed

Reply:
Address this for MVP

Reply:
Reverted to non-null check and return.